pkg/integrations/v4/logs/cfg.go

```diff
@@ -296,6 +321,18 @@ func NewFBConf(loggingCfgs LogsCfg, logFwdCfg *config.LogForward, entityGUID, ho
 		}
 	}
 
+	if totalFiles > fbFileWatchLimit {
+		warningDetails := TooManyFilesWarning{
+			TotalFiles: totalFiles,
+			LogsCfg:    loggingCfgs,
+		}
+		warningMessage, err := warningDetails.Format()
+		if err != nil {
+			return FBCfg{}, err
+		}
```

Comment:
We should not prevent the Infra-Agent to start if for some reason we have a formatting error here. Please do the following instead:
```
if err == nil {
    cfgLogger.Warn(warningMessage)
}
```

BTW, you can use the shorter syntax:
```
if warningMessage, err := warningDetails.Format(); err == nil {
    cfgLogger.Warn(warningMessage)
}
```